ecosystem/typescript/sdk_v2/src/bcs/types.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+export type Uint8 = number;
+export type Uint16 = number;
+export type Uint32 = number;
+export type Uint64 = bigint;
+export type Uint128 = bigint;
+export type Uint256 = bigint;
+export type AnyNumber = bigint | number;
```

Comment:
Are we skipping the alias from sdk v1 `export type Bytes = Uint8Array;`? In this case, we will be using `Uint8Array` instead of `Bytes` on sdk v2?

Reply:
Yeah, we talked about it a bit back that we wanted to get rid of the reference to `Bytes`. It was in a random slack thread, lol. It is easy to add back in, but we don't really use it anywhere else, so I removed it for now. It's fairly easy to add back in

Reply:
Also with some of this stuff, I am playing around with different types of serializtaion/deserialization. The core BCS serialization/deserialization should generally stay the same, though

Reply:
Cool! I am gonna start rebasing some of my stuff on top of this branch 😄

Reply:
Ok cool :) Lmk if something blocks you or is confusing. I kept most of the same file structure even